xio/fs/mkembeddedfs/main.go

```diff
@@ -96,7 +96,7 @@ func main() {
 	failIfErr(tmpl.Execute(&buffer, &cfg))
 	d, err := format.Source(buffer.Bytes())
 	failIfErr(err)
-	failIfErr(ioutil.WriteFile(output, d, 0644)) //nolint:gosec // these are the permissions I want
+	failIfErr(ioutil.WriteFile(output, d, 0600))
```

Comment:
Another case where I'd prefer the default to be 0644. Open to an option on the cmdline to restrict it further, though.